planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -38,10 +38,18 @@ function initAutocomplete() {
     let options = {
         types: ['geocode']
     };
-    autocompleteStart = new google.maps.places.Autocomplete(startAddress,options); 
+    autocompleteStart = new google.maps.places.Autocomplete(startAddress,options);
+    autocompleteStart.setFields(['address_component']);
+    autocompleteStart.addListener('place_changed', updateStartingAddress);
+
     autocompleteEvent = new google.maps.places.Autocomplete(eventAddress,options);
```

Comment:
Will you need similar listeners and fields for |autocompleteEvent| as well?

Reply:
When I tested it it seems like the input values are automatically changed with autocomplete. I only need the listener for autocompleteStart since we need to store the start location in sessionStorage